### DISCUSSION TITLE
Adding files under lua/plugins/*.lua errors out.

### DISCUSSION THREAD

**Discussion by mvs2** [LazyVim/LazyVim#333] [Q&A] [answered]
Hello,
I am new to all this, so perhaps I am making a simple mistake.  And thank you for all of this as it is awesome.  Trying to do exactly what is laid out in the [plugins doc. ](https://www.lazyvim.org/configuration/plugins) I created lua/plugins/disabled.lua and copied:

`-- disable trouble
{ "folke/trouble.nvim", enabled = false }`

Error: 
 ....local/share/nvim/lazy/lazy.nvim/lua/lazy/core/cache.lua:135: /home/marcus/.config/nvim/lua/plugins/disabled.lua:2: unexpected symbol near '{'  
    # stacktrace:                                                                                                                                             
      - ~/.config/nvim/lua/config/lazy.lua:9                                                                                                            
     - ~/.config/nvim/init.lua:2                                                                                                                               
Thanks in advance for any help.

**Comment by dpetka2001** [comment]
Please post your whole config of the file in question.

**Comment by dpetka2001** [comment, marked as answer]
The following should do the trick 
```lua
return {
  -- disable trouble
  { "folke/trouble.nvim", enabled = false },
}
```
in your `lua/plugins/disabled.lua` file.

**Reply by mvs2** [reply]
That did do the trick!  Thank you very much!

**Comment by NubiaMeldrumDN** [comment]
It looks like the error is caused by an unexpected symbol near the curly braces in the second line of the code you copied into disabled.lua. Make sure that the code is exactly as shown in the plugins documentation and that there are no extra or missing characters. If the issue persists, you can try commenting out the entire line or removing the curly braces to see if that resolves the error.

**Comment by haydonryan** [comment]
A lot of the examples don't have return at the start. this is needed in plugin files (this bit me too)
